pydatalab/tests/server/test_blocks.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+
+from pydatalab.apps import BLOCK_TYPES, BLOCKS
+
+
+@pytest.mark.dependency()
+def test_get_all_available_block_types():
+    """Test that we can enumerate all available block types."""
+    assert len(BLOCKS) > 0
+    assert len(BLOCK_TYPES) > 0
+
+    for block_class in BLOCKS:
+        assert hasattr(block_class, "blocktype")
+        assert hasattr(block_class, "name")
+        assert hasattr(block_class, "description")
```

Comment:
```suggestion
        assert block_class.blocktype is not None
        assert block_class.name is not None
        assert block_class.description is not None, f"{block_class.blocktype} is missing a description"
```

Marginal improvement; I think the hasattr will pass for anything that inherits from the base block by default, this checks that a description is properly provided.